src/PositionManager.sol

```diff
@@ -197,7 +199,7 @@ contract PositionManager is IPositionManager, ERC721Permit, PoolInitializer, Mul
     /// @param recipient the receiver of the excess native tokens. Should be the caller, the one that sent the native tokens
     function _sweepNativeToken(address recipient) internal {
         uint256 nativeBalance = address(this).balance;
-        if (nativeBalance > 0) address(recipient).call{value: nativeBalance}("");
+        if (nativeBalance > 0) recipient.safeTransferETH(nativeBalance);
```

Comment:
should we just revert if 0 instead?

Reply:
no if it’s 0 that means the sent in the perfect amount of ETH - this is just a refund